packages/frontend/src/components/UserSettings/UserAttributesPanel/index.tsx

```diff
@@ -0,0 +1,243 @@
+} from '@mantine/core';
+import {
+    IconAlertCircle,
+    IconCircleX,
+    IconEdit,
+    IconInfoCircle,
+} from '@tabler/icons-react';
+import { FC, useState } from 'react';
+import { useOrganization } from '../../../hooks/organization/useOrganization';
+import { useTableStyles } from '../../../hooks/styles/useTableStyles';
+import {
+    useUserAttributes,
+    useUserAttributesDeleteMutation,
+} from '../../../hooks/useUserAttributes';
+import { useApp } from '../../../providers/AppProvider';
+import LoadingState from '../../common/LoadingState';
+import MantineIcon from '../../common/MantineIcon';
+import { SettingsCard } from '../../common/Settings/SettingsCard';
+import ForbiddenPanel from '../../ForbiddenPanel';
+import UserAttributeModal from './UserAttributeModal';
+
+const UserListItem: FC<{
+    orgUserAttribute: OrgAttribute;
+    allUserAttributes: OrgAttribute[];
+}> = ({ orgUserAttribute, allUserAttributes }) => {
+    const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
+    const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+
+    const { mutate: deleteUserAttribute } = useUserAttributesDeleteMutation();
+
+    return (
+        <>
+            <tr>
+                <td width={800}>
+                    <Flex justify="space-between" align="center">
+                        <Stack spacing="xs">
+                            <Group spacing="two">
+                                <Text>{orgUserAttribute.name}</Text>
+                                {orgUserAttribute.description && (
+                                    <Tooltip
+                                        multiline
+                                        maw={300}
+                                        withArrow
+                                        label={orgUserAttribute.description}
+                                    >
+                                        <MantineIcon
+                                            icon={IconInfoCircle}
+                                            color="gray.6"
+                                        />
+                                    </Tooltip>
+                                )}
+                            </Group>
+                            <Text fz="xs" color="gray.6">
+                                {' '}
+                                {orgUserAttribute.users.length} user
+                                {orgUserAttribute.users.length != 1 ? 's' : ''}
+                            </Text>
+                        </Stack>
+                    </Flex>
+                </td>
+                <td>
+                    <Flex justify="space-between" align="right">
+                        <Button
+                            variant="subtle"
+                            onClick={() => setIsEditDialogOpen(true)}
+                        >
+                            <MantineIcon
+                                icon={IconEdit}
+                                size="xlg"
```

Comment:
(note for self - check this in preview env)

Reply:
'xlg' is not necesary, you can make this variant="outline" like the second button
<img width="470" alt="CleanShot 2023-07-17 at 19 44 42@2x" src="https://github.com/lightdash/lightdash/assets/962095/f7274521-e36e-4831-9c38-51c3b10b749f">

also, change `<Flex justify="space-between" align="right">`
to just `<Group>` to get a space between two icons

Reply:
mmm, group does not keep it in the same line, perhaps because of the width=1% hack ? 

![image](https://github.com/lightdash/lightdash/assets/1983672/ee90e281-fa52-4f2c-9f63-3e59e7f7c74c)

Reply:
Button variant outline (instead of subtle) makes it different :thinking: 

![image](https://github.com/lightdash/lightdash/assets/1983672/b24dc352-b239-4ad0-9fc9-45de78e95019)

Reply:
what about `<Group noWrap>`